localclient/terminal_player.go

```diff
@@ -383,10 +394,16 @@ func (tc *terminalClient) shouldCreateGame() bool {
 }
 
 func (tc *terminalClient) createGame() error {
-	opID := tc.getPlayerID("What's your opponent's username?")
-	url := fmt.Sprintf("/create/game/%s/%s", opID, tc.me.ID)
+	opID := tc.getPlayerID(`What's your opponent's username?`)
+	gameReq := model.Game{
+		Players: []model.Player{{
+			ID: opID,
+		}, {
+			ID: tc.me.ID,
+		}},
+	}
 
-	respBytes, err := tc.makeRequest(`POST`, url, nil, nil)
+	respBytes, err := tc.makeJSONBodiedRequest(`POST`, `/create/game`, gameReq)
```

Comment:
:oof: this should use a new "create game" struct instead of reusing the `model.Game` struct